scripts/vm/hypervisor/kvm/setup_agent.sh

```diff
@@ -224,17 +224,5 @@ then
     setenforce 0
 fi
 
-which aria2c
-if [ $? -gt 0 ]
-then
-    yum install epel-release -y
-    yum install aria2 -y
-    if [ $? -gt 0 ]
-    then
-        printf "failed to install aria2"
```

Comment:
fyi - this is removed as installing cloudstack-agent will install aria2 and this script assumes a yum-enabled system that will fail on debian/ubuntu etc.